go/vt/vtgate/planbuilder/testdata/postprocess_cases.json

```diff
@@ -258,6 +258,57 @@
       ]
     }
   },
+  {
+    "comment": "Test that LIMIT can be pushed to the route even in the presence of an outer join",
+    "query": "SELECT user_extra.`id` FROM user LEFT JOIN user_extra ON user_extra.`b` = 2 AND user.`c` = user_extra.`c` WHERE user.`a` = 1 LIMIT 1",
+    "plan": {
+      "QueryType": "SELECT",
+      "Original": "SELECT user_extra.`id` FROM user LEFT JOIN user_extra ON user_extra.`b` = 2 AND user.`c` = user_extra.`c` WHERE user.`a` = 1 LIMIT 1",
+      "Instructions": {
+        "OperatorType": "Limit",
+        "Count": "1",
+        "Inputs": [
+          {
+            "OperatorType": "Join",
+            "Variant": "LeftJoin",
+            "JoinColumnIndexes": "R:0",
+            "JoinVars": {
+              "user_c": 0
+            },
+            "TableName": "`user`_user_extra",
+            "Inputs": [
+              {
+                "OperatorType": "Route",
+                "Variant": "Scatter",
+                "Keyspace": {
+                  "Name": "user",
+                  "Sharded": true
+                },
+                "FieldQuery": "select `user`.c from `user` where 1 != 1",
+                "Query": "select `user`.c from `user` where `user`.a = 1",
```

Comment:
Shouldn't we be able to also push the `LIMIT 1` condition into this query here? 🤔